pwiz_tools/Skyline/SettingsUI/PeptideSettingsUI.cs

```diff
@@ -82,10 +82,16 @@ public class QuantificationTab : TabWithPage { }
         private readonly LabelTypeComboDriver _driverLabelType;
         private static readonly IList<int?> _quantMsLevels = ImmutableList.ValueOf(new int?[] {null, 1, 2});
         private readonly LabelTypeComboDriver _driverSmallMolInternalStandardTypes;
+        private readonly string _staticModsOriginalTooltip;
+        private readonly string _heavyModsOriginalTooltip;
+        private readonly string _librariesOriginalTooltip;
 
         public PeptideSettingsUI(SkylineWindow parent, LibraryManager libraryManager, TABS? selectTab)
         {
             InitializeComponent();
+            _staticModsOriginalTooltip = helpTip.GetToolTip(listStaticMods);

```

Comment:
What is this about? Needs some commenting at least. When there is no tip to be show, you are going to show the entire list?